ivy_tests/test_ivy/test_frontends/test_mindspore/test_ops/test_function/test_mindspore_nn_func.py

```diff
@@ -360,3 +360,49 @@
 #         count_include_pad=count_include_pad,
 #         divisor_override=None,
 #     )
+
+
+# @handle_frontend_test(
+#     fn_tree="mindspore.ops.function.nn_func.smooth_l1_loss",
+#     dtype_and_x=helpers.dtype_and_values(
+#         available_dtypes=helpers.get_dtypes("float"),
+#         num_arrays=2,
+#         allow_inf=False,
+#         shared_dtype=True,
+#     ),
+#     size_average=st.booleans(),
+#     reduce=st.booleans(),
+#     reduction=st.sampled_from(["none", "mean", "sum"]),
+#     beta=st.sampled_from([1.0, 0.5, 0.1, 0.0]),
```

Comment:
Your code says `beta=0` isn't allowed, so I don't know why it's in this list. Also, would it be better to choose from `st.floats(min_value=0.1)` to get a wider range?

Reply:
and have kept zero because to show value error...as in main functions also ,there have raised value error